Data/db_postgresql.py

```diff
@@ -113,21 +111,20 @@
             cache[place]['cityName'],
             cache[place]['stateCode'],
             cache[place]['stateName'],
-            cache[place]['county'],
             cache[place]['lat'],
             cache[place]['lon'],
             cache[place]['counter']])
 
     sql = """
         INSERT INTO places
-        (city, state_code, state_name, county, latitude, longitude, counter)
+        (city, state_code, state_name, latitude, longitude, counter)
         VALUES %s
         """
     psycopg2.extras.execute_values(
         pg_curs, sql, params, page_size=10000)
 
 # function that inserts data into tables from training data csv
-with open('/Users/michelle/Labs25-Human_Rights_First-TeamC-DS/Data/training_data.csv', 'r') as f:
+with open('training_data.csv', 'r') as f:
     reader = csv.reader(f)
```

Comment:
did you check to see if this path works? You may need to reference it differently if we move these files around.